### PR TITLE
#21 Explain how to exit full screen

### PR DESCRIPTION
**Pull request by gscdev**

### CODE REVIEW

**Comment by jogrimst** [comment]
@sladomic Can this pull request be merged now?

**Comment by sladomic** [comment]
@jogrimst shouldn't we first revert to the code before this pull request https://github.com/literacyapp-org/literacyapp-start-guide/pull/20 and then we can merge this one?

**Comment by jogrimst** [comment]
> @jogrimst shouldn't we first revert to the code before this pull request #20 and then we can merge this one?

@sladomic Yes, you are right. I created this issue: #22

File: app/src/main/java/org/literacyapp/startguide/content/swipe/ExitFullScreenActivity.java [hunk 1]
```diff
@@ -73,13 +74,13 @@ private void hideSystemBars() {
     }
 
     private void playExitFullScreen() {
-        // TODO: 04/05/2017 audio move bottom to exit full screen
-//        MediaPlayerHelper.playWithDelay(this, R.raw.exit_full_screen, new MediaPlayerHelper.MediaPlayerListener() {
-//            @Override
-//            public void onCompletion() {
+        // TODO: 04/05/2017 audios en/sw Slide down from outside the screen
+        MediaPlayerHelper.playWithDelay(this, R.raw.slide_down_from_outside_the_screen, new MediaPlayerHelper.MediaPlayerListener() {
```
Comment (jogrimst) [inline app/src/main/java/org/literacyapp/startguide/content/swipe/ExitFullScreenActivity.java]: @gscdev I'm preparing these audios now.

**Pull request merged**